app/models/miq_request.rb

```diff
@@ -200,7 +200,7 @@ def call_automate_event_queue(event_name)
       :method_name => "call_automate_event",
       :args        => [event_name],
       :zone        => options.fetch(:miq_zone, my_zone),
-      :msg_timeout => 3600
+      :msg_timeout => options[:msg_timeout].presence || 3600
```

Comment:
I don't think this is going to work, this is the msg_timeout for this one queue message but VM provisioning is a state_machine and each state is its own queue message.  E.g. in the BZ the timeout was 600 sec which is the default queue_worker timeout.

Reply:
Aside from @agrare's comment, at a minimum we also have to clamp this to valid values.  Since it's available through the dialog, *any* number could end up in there in there (and possible even raw text since it can be set through automate).  So, at a minimum we need something like:

```ruby
options[:msg_timeout].presence || 3600).to_i.clamp(0..3600)
```